services/sleeper/docker-compose-meta.yml

```diff
@@ -2,50 +2,45 @@ services:
   sleeper:
     build:
       labels:
-        io.simcore.authors:
-          '{"authors": [{"name": "Andrei Neagu", "email": "neagu@itis.swiss",
+        io.simcore.authors: '{"authors": [{"name": "Odei Maiz", "email": "maiz@itis.swiss",
+          "affiliation": "IT''IS Foundation"}, {"name": "Andrei Neagu", "email": "neagu@itis.swiss",
```

Comment:
Manuel was the original author of sleeper 1.0.0 I think...

Reply:
he was also added